test/test_helper.rb

```diff
@@ -1,3 +1,18 @@
-require 'rails'
+require 'rails/railtie'
+require 'action_mailer'
 require 'aws-sdk-rails'
 require 'minitest/autorun'
+
+class TestMailer < ActionMailer::Base
+  layout nil
+
+  def deliverable(body:, from:, subject:, to:)
```

Comment:
Please use an 'opts' hash rather than keyword args here. We still maintain test compat with older Ruby versions, for customers with LTS of deprecated versions, in this library. Those tests are breaking.

Reply:
No worries, that’s done.